contract.md

```diff
@@ -12,29 +14,41 @@ like to work on, as part of your project (for example, other things related to D
 you have come across, or topics from other classes)? Why are you interested in working on
 these things?_
 
+_I want to further work on using extensions because I feel like there were some issues I ran into with using extensions in the last homework assignment, and I feel like extensions have more capabilities than what I used it for. I also want to continuing working on writing an internal langauge because I feel like that applies that most concepts that I've learned from class and homework assignments._
+
 ## Time management plan
 
 _How do you plan to set aside time outside of class, to work on the project? Are there
 intermediate milestones that you can create, to help you make consistent progress?_
 
+_Since the milestones of the project are due every Sunday and the feedback is due every Tuesday, I plan on starting work for the milestones either during or after class on Tuesday, work on it slightly again during or after class on Thursday. Since I have a lot of free time on Fridays, I feel like I'll have time to finish up a lot of work on Fridays. I think I'm mostly going to follow the milestones that are already set for me with the weekly homework assignments._
+
 ## Teamwork plan
 
 _If you plan to work with someone else on the project, what is your plan for
 collaborating? What part(s) do you think you might do together? What part(s) are you
 considering doing separately? How will you hold one another accountable to make regular
 progress?_
 
+_I don't plan on working wiht a partner on my project because I tend to lose interest on a project when I have to compromise on what the topic of the project is._
+
 ## Critique plan
 
 _What will you do to make sure that you can give consistent and actionable feedback to
 other people in the class?_
 
+_I'll follow the feedback deadlines carefully and make sure to give people feedback soon after I finish my own milestone of the project. This will help me remember what had to be done for the milestone, so I'll be able to give better feedback._
+
 ## Success
 
 _At the end of the semester, what would you be proud to show or tell someone about your
 project?_
 
+_I would be proud to show them that I've learned a lot about my domain. At the moment I'm considering writing a language that is similar to javascript, but is specfically used for videos. I want to be able to at least learn more about how javascript is used for videos._
+
 ## Assessment
 
 _Looking over your responses to the previous questions, what would you consider to be an
 "A-" for your project? What would you consider to be an "A"?_
+
+_I would consider an "A-" to be a language that can manipulate a video. I would consider an "A" to be a language that could pretty much replace javascript for editing videos._
```

Comment:
To me, that sounds like a pretty high bar for an A since I imagine the JS tool took years to develop, so it is unlikely you can replace all the functionality in a few weeks.